libraries/RC_Channel/RC_Channels.cpp

```diff
@@ -103,7 +103,7 @@ const AP_Param::GroupInfo RC_Channels::var_info[] = {
     // @User: Advanced
     // @Range: 0.0 120.0
     // @Units: s
-    AP_GROUPINFO("_OVERRIDE_TIME", 32, RC_Channels, _override_timeout, 3.0),
+    AP_GROUPINFO("_OVERRIDE_TIME", 32, RC_Channels, _override_timeout, -1),
```

Comment:
This looks unrelated.

Reply:
This parameter can also cause catastrophic results with R/C overrides since it changes the historic behavior of overrides. The GCS is forced to send R/C overrides before the timeout expires. This was not the case in the past and their has been no formal requirement that I know of for GCSes to continually send R/C override requests, especially when the state has not changed.

Reply:
Surely we should have the timeout on by default and I think all ground stations do constantly send the rc-overrides if they're using them.

If we set this timeout to -1 does the RC failsafe still trigger OK?

Reply:
The GCS needs to always be sending overrides anyways since it has no clue what ones have made it to the vehicle or not, and if it fails to keep sending them the RC failsafe could be triggered anyways.

Reply:
@WickedShell, but if the failsafe is going to catch the problem anyway then it makes sense that we should be able to remove this timeout doesn't it?  It would then be similar to a regular receiver that is setup to stop-sending-pulses... with these receivers we maintain the last value they sent but we also trigger a failsafe.

Reply:
I discussed this on mumble with @WickedShell and this parameter controls how long it takes until AP falls back to the regular transmitter.  So if this param is set to -1 it will never fallback which is fine if there is no transmitter but not good if there is a transmitter.  I think it's safer to leave the parameter's default at 3.